aries-site/src/examples/components/pageheader/PageHeaderResponsiveActions.js

```diff
@@ -7,10 +7,29 @@ import {
   Page,
   PageContent,
   ResponsiveContext,
+  ThemeContext,
 } from 'grommet';
 import { groupActions } from './utils';
 import { ActionMenu, ReverseAnchor } from '../../templates';
 
+// need to overrid the PageHeader theme values in order for our examples 
+// to fit their container page. Even though it is using the correct breakpoints
+// our examples are only taking up half the space on the page so layout different than
+// a PageHeader on a full page.
+const PageHeaderSiteTheme = {
```

Comment:
instead of repeating this in 2 places, can we define it in a single file and import to the 2 examples?

Reply:
one is a little different than the other as it contains `medium` for the dos and don't example since those have a smaller width I had to change them

Reply:
oh gotcha, okay that's fine. can we adjust the capitalization to `pageHeaderSiteTheme` here too?

Reply:
done